Docs/source/Hardware/Boards.rst

```diff
@@ -8,6 +8,231 @@ Each board measures up to 12 battery modules and 16 temperature sensors, but is
 
 The Amperes minion board is connected to the main power cable that powers the electrical loop using the :term:`Shunt Resistor`. 
 
+Display Board
+=============
+
+Overview
+^^^^^^^^
+`GitHub Link <https://github.com/lhr-solar/BPS-DisplayPCB>`__
+
+`BOM Link <https://www.mouser.com/ProjectManager/ProjectDetail.aspx?State=EDIT&ProjectGUID=11b675c5-b15b-46bd-a790-bb450819a4d3>`__ 
+
+Brief Description/Purpose:
+    The purpose of this board is to display messages from the :term:`CAN <CAN Bus>` interface on 
+    the E-Ink display for debugging.
+
+Pertinent Regulations
+^^^^^^^^^^^^^^^^^^^^^
+========== ============================================== ===============================================
+Regulation Description of Regulation                      How Regulation is Met
+
+8.3.B.2    | All supplemental batteries must have at a    The "SUPP CHECK" (Supplemental Check) LED
+           | minimum Passive Protection for under voltage 
+           | where charging occurs remote to the solar 
+           | vehicle unless they are primary cells. 
+           | Active Protection is required if charging is 
+           | within the solar vehicle. No Secondary 
+           | Lithium battery types shall be used for the 
+           | Supplemental Battery unless the Supplemental 
+           | Battery is powering a commercially procured 
+           | component such as a cell phone or laptop and 
+           | the Supplemental Battery was intended for 
+           | this purpose.
+========== ============================================== ===============================================
+
+Context
+^^^^^^^
+**Location of the Board:** On a movable arm mount attached to the south end of the board. Mounted near the driver.  
+
+List of I/O and Connections: 
```

Comment:
Some of the titles for bulleted lists are bold and italicized, some are just italicized, and this one is neither. Can we make them consistent?

Reply:
Yea, now that I think about it it's kinda weird. Sometimes if I do *word* it makes it bold but other times I have to do **word**. \_(--)_/

Reply:
No way, you can **bold** th**i**ngs in github comments.

Reply:
AAND *ITALICIZE* WHHAAAATTT

Reply:
Yeah, you can do a lot in github comments. [You can even add images](https://github.com/lhr-solar/BPS/issues/207#issuecomment-699601773)